### PR TITLE
feature / create accredited page in prismic repeatable

### PR DESCRIPTION
**Pull request by sopoour**
* created the repeatable custom type with the respective fields as found in Linear [here](https://linear.app/bankgreen/issue/GROOT-209/create-accredited-page-in-prismic-repeatable)
* created a very broad setup for the page in nuxt where the prismic data can be used and the sections added

### CODE REVIEW

File: pages/accreditation/[slug].vue [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+    <section class="bg-gradient-to-b from-sushi-50 to-sushi-100 md:pt-28 max-w-screen ">
+      <div class="page-fade-in contain flex flex-col justify-center items-center max-w-8xl pt-28 pb-20 md:pb-0 lg:pt-12 gap-10">
+        <h1>{{ accred?.data.hero_title || "Your money is building a greener future" }}</h1>
+      </div>
+      <Swoosh />
+    </section>
+  </div>
+</template>
+
+<script setup lang="ts">
```
Comment (sopoour) [inline pages/accreditation/[slug].vue:12]: this whole thing is just a draft right now. The sections will be added with the upcoming tickets

File: pages/accreditation/[slug].vue [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+/* import { asImageSrc } from '@prismicio/helpers'
+import { defineSliceZoneComponents } from '@prismicio/vue'
+import { components } from '~~/slices' */
+
+const route = useRoute()
+/* const error = ref(false) */
+
+const { client } = usePrismic()
+const slug = route.path.split('/').at(-1) ?? ''
```
Comment (claudiabdm) [inline pages/accreditation/[slug].vue:21]: Just a silly thing here 😅 I usually access to the slug with `route.params.slug`. Like [here](https://nuxt.com/docs/getting-started/routing#route-parameters). Is there a reason why is `route.path.split('/').at(-1)` instead?

**Review by claudiabdm — COMMENTED** [review]
Thanks! Looks good 👍

Just wrote two little comments.

File: customtypes/accreditationpage/index.json [hunk 3]
```diff
@@ -0,0 +1,116 @@
...
+      },
+      "hero_logo": {
+        "type": "Image",
+        "config": { "label": "Hero Logo", "constraint": {}, "thumbnails": [] }
+      },
+      "projects": {
+        "type": "Group",
+        "config": {
+          "label": "Projects",
+          "fields": {
```
[the remark this answers is not in the document]
Comment (sopoour) [reply customtypes/accreditationpage/index.json:31]: good point, thanks!

**Pull request merged**